SeaPublicWebsite/Controllers/EnergyEfficiencyController.cs

```diff
@@ -1154,11 +1155,12 @@ public IActionResult YourRecommendations_Get(string reference)
             ).ToList();
             userDataStore.SaveUserData(userDataModel);

```

Comment:
In the code above here we take a `Recommendation` object and turn it into a `UserRecommendation` object with exactly the same fields. Can we scrap one of those classes?

Reply:
Ok, fair enough. Maybe rename `Recommendation` to `BreRecommendation` and move it into the Bre folder though so it's more clear what the distinction is.